lib/kv-redis.js

```diff
@@ -151,7 +151,9 @@ RedisKeyValueConnector.prototype.disconnect = function(cb) {
 RedisKeyValueConnector.prototype.set =
 function(modelName, key, value, options, callback) {
   const self = this;
-  ModelKeyComposer.compose(modelName, key, function(err, composedKey) {
+  const customModelName = self.modelName(modelName);
```

Comment:
I find `customModelName` misleading, because in most cases the name is not going to be custom. Have you considered re-using the same variable?

```ts
modelName = self.modelName(modelName);
```

If you prefer to keep the data immutable, then I am proposing to use variable name `resolvedModelName`.